profiles/apparmor.d/usr.share.openqa.script.worker

```diff
@@ -124,10 +124,11 @@
   /var/lib/openqa/share/factory/hdd/ r,
   /var/lib/openqa/share/factory/hdd/* r,
   /var/lib/openqa/share/factory/hdd/fixed/* r,
-  /var/lib/openqa/share/factory/iso/* r,
+  /var/lib/openqa/share/factory/iso/* rk,
   /var/lib/openqa/share/factory/iso/fixed/* r,
   /var/lib/openqa/share/factory/other/* r,
   /var/lib/openqa/share/tests/** r,
+  owner /sys/**/ rw,
```

Comment:
See https://doc.opensuse.org/documentation/leap/security/html/book.security/cha.apparmor.profiles.html#sec.apparmor.profiles.perm.owner
It is a conditionnal to restrict to user who created the file.
Otherwise, we can just set  to */sys/**/ rw*  which is currently read-only.

Reply:
Thanks for teaching me a thing - this looks really useful.